pnpm-lock.yaml

```diff
@@ -8556,7 +8556,6 @@ packages:
     dependencies:
       is-hex-prefixed: 1.0.0
       strip-hex-prefix: 1.0.0
-    bundledDependencies: false
```

Comment:
Not declaring `bundleDependencies` is [effectively](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bundledependencies) `false`